terraform/modules/forest_node/user-data.tpl

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+NEW_USER="${NEW_USER}"
```

Comment:
you don't need to quote these vars. Also, why are you doing this assignment?

Reply:
as in, you're assigning a variable to itself, which is a no-op, you can remove all lines like this